projects/plugins/boost/app/lib/minify/functions-helpers.php

```diff
@@ -134,11 +88,11 @@ function jetpack_boost_page_optimize_js_exclude_list() {
 
 function jetpack_boost_page_optimize_js_exclude_list_default() {
 	// WordPress core stuff, a lot of other plugins depend on it.
-	return array( 'jquery', 'jquery-core', 'underscore', 'backbone' );
+	return Minify_JS::$default_excludes;
 }
 
 function jetpack_boost_page_optimize_css_exclude_list() {
-	$exclude_list = get_option( 'page_optimize-css-exclude' );
+	$exclude_list = jetpack_boost_ds_get( 'minify_css_excludes' );
 	if ( false === $exclude_list ) {
```

Comment:
Are we ever going to return false from the exclude list in boost?

Reply:
Aw man, I hadn't pushed my changes. Here's the latest version - https://github.com/Automattic/jetpack/pull/30005/commits/f067baa753afcdc44f4781edb0d24003434cd469.